### ISSUE TITLE
Update Amazon (retail) info

### ISSUE THREAD

**Issue by kstlouis** [2factorauth/twofactorauth#2841]
Amazon retail on your site shows a warning that it requires SMS or a phone call to set up, which is not (or at least no longer) true. Just set it up in a third-party app with no SMS at all, though SMS is still the default.

Also, looks like phone call is available as an alternative to SMS, so that should show as an option.

**Comment by Jawshy** [comment]
Hi, @kstlouis. Amazon's [Turning on Two-Step Verification](https://www.amazon.com/gp/help/customer/display.html?nodeId=201962420) help page says: 

> You won't be able to turn on Two-Step Verification without adding a back-up phone number .

Is their documentation inaccurate?

**Comment by kstlouis** [comment]
It would appear to be. I currently have it set up via an authenticator app, and do not have a phone number on my account at all.

**Comment by kstlouis** [comment]
*closed issue by accident

**Comment by stephengroat** [comment]
confirmed that phone is not required with amazon today

they said they would change docs, we'll see if that ever happens

**Comment by elhossari** [comment]
I tried doing removing the SMS method from my account today and found myself in a weird state where 2FA *appeared* to be enabled but wasn't actually.

Here's what that looked like on the "Your Account > Login & security > Advanced Security Settings" page:
![image](https://user-images.githubusercontent.com/1196102/36645402-d5332534-1a1c-11e8-8b26-2d6011101656.png)

In this state, I wasn't prompted for a code when logging in.

And it wasn't until I added a backup number for SMS that it actually did so. After that, the same page has a new section at the top that shows a status of "Enabled":
![image](https://user-images.githubusercontent.com/1196102/36645455-f11a3296-1a1d-11e8-8013-63a23bc7af04.png)

So unfortunately, it looks like when 2FA settings are configured but not enabled, the status doesn't appear at all, which is confusing.

@kstlouis, are you prompted for the TOTP code on login?

**Comment by kstlouis** [comment]
I am now! 
I feel pretty silly, but it looks like 2FA was never enabled on my account. I had set up an authenticator, but (as the documentation accurately reflects), it was never enabled as I had not set up the required backup phone number. I didn't notice because I'm never prompted for a password on my devices (I have auto sign-in enabled on the 2 devices I only ever use for Amazon).

I spent all this time thinking that 2FA had been correctly configured when I had actually only completed half of the setup process. (My bad..)
